airflow/current/entrypoint.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+function init_gcloud {
+  EMAIL=${GCP_AIRFLOW_ACCOUNT_EMAIL:?"Need to be set and non-empty"}
+  FILE=${GCP_AIRFLOW_SA_FILE_PATH:?"Need to be set and non-empty"}
+  LOCATION=${GCP_K8S_LOCATION:?"Need to be set and non-empty"}
+  ZONE=${GCP_K8S_ZONE:?"Need to be set and non-empty"}
+  PROJECT=${GCP_K8S_PROJECT:?"Need to be set and non-empty"}
+  
+  #Start gcloud auth
```

Comment:
Can we standarize comments to `# comment`?

Reply:
yes. 👍